.github/workflows/pr-management.yml

```
@@ -0,0 +1,89 @@
+      - name: Cleanup Merged Branches
+        if: github.event_name != 'pull_request'
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const branches = await github.rest.repos.listBranches({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              protected: false
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)